core/common/src/test/java/alluxio/security/authorization/ModeTest.java

```diff
@@ -163,4 +163,46 @@ public void umaskNotIntegerTest() {
         PropertyKey.SECURITY_AUTHORIZATION_PERMISSION_UMASK));
     Mode.getUMask();
   }
+
+  /**
+   * Tests the {@link Mode#setOwnerBits(Mode.Bits)} method.
```

Comment:
We could remove these javadocs - they duplicate the name of the test, and readers can go to the definition of the `setOwnerBits` calls so they don't need the link.